Content.Shared/Ame/Components/AmeFuelContainerComponent.cs

```diff
@@ -16,4 +16,9 @@ public sealed partial class AmeFuelContainerComponent : Component
     /// </summary>
     [DataField, ViewVariables(VVAccess.ReadWrite), AutoNetworkedField]
     public int FuelCapacity = 500;
+
+    // DeltaV start
```

Comment:
use this instead of start/end for a single field
```cs
/// <summary>
/// DeltaV - Extra price when sold per unit of fuel remaining.
/// </summary>
```